.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,176 @@
+name: integration-tests
+on:
+  push:
+    branches: [ "518-implement-semi-automatic-integration-tests" ] #REMOVE WHEN ON MASTER 
+  #pull_request_review:
+  #  types: [submitted]
+  #  branches: [ "518-implement-semi-automatic-integration-tests" ]
+  workflow_dispatch:
+    inputs:
+      bob_amount_of_nodes:
+        description: 'Amount of nodes in bob cluster for tests'
+        required: true
+        default: '4'
+      bob_disks:
+        description: 'Vdisks count'
+        required: true
+        default: '4'
+      bob_replicas:
+        description: 'Amount of vdisks replicas'
+        required: true
+        default: '2'
+      bob_payload:
+        description: 'Payload size'
+        required: true
+        default: '10240'
+      bob_count:
+        description: 'Amount of records to process during test'
+        required: true
+        default: '10000'
+
+jobs:
+  build-binaries:
+    # if: github.event.review.state == 'approved'
```

Comment:
This is incorrect. You should run job when this is not review submit or when it is an approved review submit. Thus you should add a check for event type